packages/forms/resources/views/components/radio.blade.php

```diff
@@ -54,16 +58,18 @@
                                     'border-danger-600 ring-1 ring-inset ring-danger-600' => $errors->has($getStatePath()),
                                     'dark:border-danger-400 dark:ring-danger-400' => $errors->has($getStatePath()) && config('forms.dark_mode'),
                                 ]) }}
-                                {!! ($isDisabled || $isOptionDisabled($value, $label)) ? 'disabled' : null !!}
+                                {!! $shouldOptionBeDisabled ? 'disabled' : null !!}
                                 wire:loading.attr="disabled"
                             />
                         </div>
 
                         <div class="text-sm">
                             <label for="{{ $getId() }}-{{ $value }}" @class([
                                 'font-medium',
-                                'text-gray-700' => ! $errors->has($getStatePath()),
-                                'dark:text-gray-200' => (! $errors->has($getStatePath())) && config('forms.dark_mode'),
+                                'text-gray-700' => ! $errors->has($getStatePath()) && ! $shouldOptionBeDisabled,
```

Comment:
Please use `opacity-70` to handle the text colour, which is consistent with everything else. You also won't need to worry about dark mode, then.

Reply:
Since this opacity affects only pure text and has no background like f.e. `TextInput`, I would honestly say 70 is still far too visible for it to be recognised as `disabled`. I suggest 50 instead of 70.

opacity-70
![image](https://user-images.githubusercontent.com/33912290/226453137-fa9acc05-9350-4d9a-94c8-4d967518d38b.png)

opacity-50
![image](https://user-images.githubusercontent.com/33912290/226454152-bc9af762-7da8-4a3c-9854-0443e856e4d6.png)

Reply:
Sure, 50 looks good

Reply:
We can remove the `$shouldOptionBeDisabled` check from these text color lines I think, since they now have opacity. Maybe you'll need to increase the opacity to 70 if you were previously testing with black text.

Reply:
Yeah, I'll get rid of the excess `$shouldOptionBeDisabled` checks.
I tested it out, opacity-50 seems great with every one of them, text-gray-700, dark:text-gray-200, text-danger-600, even dark:text-danger-400 :)